src/Components/Main/AssigneeList/AssigneeList.js

```diff
@@ -3,25 +3,27 @@ import Assignee from './Assignee/Assignee';
 import './_assignee_list.sass';
 
 class AssigneeList extends Component {
-  // constructor(props) {
-  //   super(props);
-  // }
-  //
-  // componentDidMount() {
-  //   console.log('why', this.props);
-  //   fetch(`https://johariwindowapi.herokuapp.com/api/v1/users/${this.props.user.id}/assignments`)
-  //     .then(result => result.json())
-  //     .then(data => this.setState({ assignees: data }))
-  // }
+  constructor(props) {
+    super(props);
+    this.state = { assignees: [] }
+  }
+  
+  componentDidMount() {
+    console.log('why', this.props);
+    this.props.user.id && fetch(`https://johariwindowapi.herokuapp.com/api/v1/users/${this.props.user.id}/assignments`)
```

Comment:
You're right! So I brought in thunk, which allows us to use fetch calls inside our actions.